build.gradle

```diff
@@ -14,7 +14,7 @@ plugins {
     // removed this plugin and filled in specific version numbers. I also removed all
     // version numbers with `+` in them. Was:
     //     id 'io.spring.dependency-management' version '1.0.10.RELEASE'
-    id 'org.springframework.boot' version '3.1.2'
+    id 'org.springframework.boot' version '3.2.1'
```

Comment:
Any downside in moving to `3.2`? I didn't see anything that would drastically affect us.

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.2-Release-Notes